content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,230 @@ aliases:
     - /docs/concepts/security/rbac/
 ---
 
-Istio aims to enhance the security of microservices and their communication
-without requiring service code changes. It is responsible for:
+Istio offers strong security to the microservices and their communication.
+For example, traffic encryption to defend against the man-in-the-middle attack,
+mutual TLS and fine-grained access policies to provide flexible service access control
+and auditing to detect who did what at what time.
 
-- Providing each service with a strong identity that represents its role to
-  enable interoperability across clusters and clouds
+This page gives an overview of how Istio security can be leveraged to secure your services, wherever you run them.
+In particular, Istio security mitigates both insider and external threats against your data, endpoints, communication and platform.
 
-- Securing service to service communication and end-user to service
-  communication
+{{< image width="80%" ratio="56.25%"
+    link="./overview.svg"
+    alt="Istio security overview."
+    caption="Istio Security Architecture"
+    >}}
+
+Istio security provides strong identity, powerful policy, transparent TLS encryption and AAA (authentication/authorization/audit)
```

Comment:
Consistency issue: Abbreviations come in parenthesis after the expanded text. There are several missing nouns in this sentence. I tried to fill in the blanks but I could not figure out what is meant to come after: strong identity, maybe control? and powerful policy, maybe configurations? I suggest:

The Istio security features provide strong identity, powerful policy, transparent TLS encryption, and authentication, authorization and audit (AAA) tools to protect your services and data.